python/ray/autoscaler/_private/aws/node_provider.py

```diff
@@ -38,6 +38,7 @@
 logger = logging.getLogger(__name__)
 
 TAG_BATCH_DELAY = 1
+LIST_RETRY_DELAY = 1
```

Comment:
nit (no merge blocker, personal preference): is it better to add units? i.e. `LIST_RETRY_DELAY_SEC`

Reply:
👍 👍